dev-vcs/guitar/guitar-0.2.0~git.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="A QT git client"
+DESCRIPTION="Git GUI Client
+
+* Open Source, Free Software
+* Run on Windows, macOS and Linux
+* Written in C++
+* Powered by Qt 5"
+HOMEPAGE="https://soramimi.github.io/Guitar/"
+COPYRIGHT="2017 S.Fuchita"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="828a6f934d86f8eb42a81c12fdb6ae52063a6835"
+SOURCE_URI="https://github.com/soramimi/guitar/archive/$srcGitRev.tar.gz"
```

Comment:
```diff
-SOURCE_URI="https://github.com/soramimi/guitar/archive/$srcGitRev.tar.gz"
+SOURCE_URI="https://github.com/soramimi/Guitar/archive/$srcGitRev.tar.gz"
```

Reply:
I **_do_** know that it also works with a lower case "g".
I'm just suggesting this to because the official name is "Guitar" (and also because we have an upper-case "G" in both `SOURCE_DIR` and `SOURCE_FILENAME`.)